agent/config/runtime.go

```diff
@@ -425,6 +425,11 @@ type RuntimeConfig struct {
 	// hcl: telemetry { dogstatsd_tags = []string }
 	TelemetryDogstatsdTags []string
 
+	// PrometheusRetentionTime enables prometheus Support for metrics if greater than 0
```

Comment:
Looking at only the description it's unclear to me why this isn't a boolean (= what does the duration do ?)